webapp/src/action_types.js

```diff
@@ -8,3 +8,4 @@ export const GET_IN_ISSUES = pluginId + '_get_in_issues';
 export const RECEIVED_SHOW_RHS_ACTION = pluginId + '_show_rhs';
 export const UPDATE_RHS_STATE = pluginId + '_update_rhs_state';
 export const SET_RHS_VISIBLE = pluginId + '_set_rhs_visible';
+export const SET_HIDE_TEAM_SIDEBAR_BUTTON = pluginId + '_set_hide_team_sidebar';
```

Comment:
I suggest `SET_HIDE_TEAM_SIDEBAR_BUTTONS` as there are multiple buttons we're hiding